src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -20,7 +20,7 @@ class AggregatorSuite extends SparkSuite {
       .annotateVariantsExpr(
         """va.test.callrate = gs.fraction(g => g.isCalled()),
           |va.test.AC = gs.map(g => g.nNonRefAlleles()).sum(),
-          |va.test.AF = gs.map(g => g.nNonRefAlleles()).stats().sum.toDouble() / gs.filter(g => g.isCalled()).count() / 2.0,
+          |va.test.AF = gs.map(g => g.nNonRefAlleles()).stats().sum.toFloat() / gs.filter(g => g.isCalled()).count() / 2.0,
```

Comment:
Question: Should we prefer explicit sizes in (Float64) in our expr code or embrace the default aliases?

I'm slightly hesitant about the aliases.  I fear they might cause more confusion than they're worth.